### ISSUE TITLE
Color: Add methods to convert between `ScottPlot.Color` and `System.Drawing.Color`

### ISSUE THREAD

**Issue by cataclism** [ScottPlot/ScottPlot#3953]
Hi @swharden, I have a suggestion to make for the color managment in ScottPlot.

The function you see below is a function that I've written to convert ScottPlot colors to a System.Drawing.Color in ARGB format.

I've done this since I had the need to color match the signal and the other elements of the interface not belonging to ScottPlot, like the checkBoxes.

Currently there is a way to assign to a ScottPlot color a System.Drawing.Color , but not the viceversa.

The usage could be of the type:

`checkBox.ForeColor=signal.Color.ToARGB();
`

Where the ToARGB() function could automatically convert the color to a string and then apply the ParseColorString as it is right now
```cs
 //Function used to extract the ARGB values from a string coming from a ScottPlot color and returns a System.Drawing.Color
 static System.Drawing.Color ParseColorString(string colorString)
 {
     // Regular expression to extract R, G, B, A values
     string pattern = @"R=(\d+), G=(\d+), B=(\d+), A=(\d+)";
     Match match = Regex.Match(colorString, pattern);

     if (match.Success)
     {
         // Convert extracted values to integers
         int r = int.Parse(match.Groups[1].Value);
         int g = int.Parse(match.Groups[2].Value);
         int b = int.Parse(match.Groups[3].Value);
         int a = int.Parse(match.Groups[4].Value);

         // Create color using FromArgb
         return System.Drawing.Color.FromArgb(a, r, g, b);
     }

     throw new ArgumentException("Input string is not in the correct format.");

 }
```

Let me know what you think about it ;)

**Comment by swharden** [comment]
Hi @cataclism, thanks for this suggestion!

This is the core statement that stuck out for me:

> Currently there is a way to convert to a `ScottPlot.Color` to a `System.Drawing.Color`, but not vice versa

I'll definitely add this feature. Thanks for your code suggestion! I'll follow-up here when it is implemented 👍

**Comment by aespitia** [comment]
put in a PR for this to convert from system.drawing.color to scottplot.color

https://github.com/ScottPlot/ScottPlot/pull/3964

**Comment by cataclism** [comment]
> put in a PR for this to convert from system.drawing.color to scottplot.color
> 
> #3964

Hi @aespitia ! Thanks for your contribution! However, what I was referring to is the opposite of what you're doing in your PR.

Infact there is the method FromColor that is used to Convert color from System.Drawing,Color to ScottPlot.Color, but not the viceversa

**Comment by aespitia** [comment]
i totally misunderstood the request, i can update the PR to go the other way

**Comment by swharden** [comment]
resolved in #3964
